### DISCUSSION TITLE
Getter for ExpirationIntent value is missing.

### DISCUSSION THREAD

**Discussion by cmeddins-e2g** [imdhemy/appstore-iap#14] [General]
Thank you and your contributors for all your work to date. We are reviewing laravel-in-app-purchases package for one of our development projects. We're currently focusing on the Apple client and so far it looks great!
We noticed recently that there is currently no public getValue() for the ExpirationIntent. Fortunately it doesn't affect our initial release but we would appreciate an update to make the value accessible.

There is already an open PR that addresses the issue:
https://github.com/imdhemy/appstore-iap/pull/12

Thanks!

**Comment by imdhemy** [comment]
Hi @cmeddins-e2g,

Thank you for your nice words and for starting this discussion. Currently, I'm working on new releases for the 3 packages which will include:

- Code refactoring and enhancement.
- Adding the missing methods and PR #12 by @R4wizard is considered.
- Supporting the new events.
- Adding deprecation messages and replacements.

And Enhancing the development experience allowing the back-end developer to complete his work without a need for purchase receipts from the mobile application.🚀

**Comment by imdhemy** [comment]
@cmeddins-e2g 

Could you please have a look at this [pull request](https://github.com/imdhemy/appstore-iap/pull/16) https://github.com/imdhemy/appstore-iap/pull/16
